### ISSUE TITLE
Doesn't work in this case

### ISSUE THREAD

**Issue by Zibri** [cowbell/sharedrop#63]
Phone A connected to someprovider
Phone B connected to someotherprovider

Since both A and B are behind NAT, sharedrop doesn't work as it is now.
But WebRTC has a mechanism for this.
All you have to do is to use a TURN server with relay.

**Comment by szimek** [comment]
I don't think it will ever be supported, unless someone can find/sponsor a free TURN server with high enough transfer limit. It would be great to at least show an error in this case, though.

**Comment by Zibri** [comment]
anyone can setup such a serve using coturn on their own computer.

Also, I have found a way to always have a turn server available, but I can't discuss it here.

**Comment by szimek** [comment]
Feel free to create a PR that add such option, but like I said - the hosted version of ShareDrop won't support it unless someone can provide a TURN server.

**Comment by Zibri** [comment]
if I want to install locally sharedrop from the source, what do I have to do? It's not clear in the readme. I just want to run a locally hosted version to do some tests.

**Comment by szimek** [comment]
I set it up a few years ago and haven't touch it since then, so I have no idea if it's going to work.

I *think* it should be enough though:
* clone the repo
* run `yarn` to install all dependencies
* run `cp .env{.sample,}` to create `.env` file
  * `SECRET` key is used to encrypt cookies and generate room name based on public IP address for / route. It can be any random string - you can generate one using e.g. `date | md5sum`
  * you need to set `FIREBASE_URL` and `FIREBASE_SECRET` as well (there's info how to create one in the `README`)
* run `yarn run dev` to start the app
